index.html

```diff
@@ -72,6 +74,18 @@
 								that this has already happened.
 							</div>
 						</div>
+						<div class="tabs-item" data-tab="5">
+							<div class="tabs-item-title">Quote 5</div>
+							<div class="tabs-item-description">
+								Be yourself; everyone else is already taken.
+							</div>
+						</div>
+						<div class="tabs-item" data-tab="6">
+							<div class="tabs-item-title">Quote 6</div>
+							<div class="tabs-item-description">
+								Two things are infinite: the universe and human stupidity; and I'm not sure about the universe.
+							</div>
+						</div>
 					</div>
 				</div>
 			</div>
```

Comment:
- Great job on the assingment Aaron. You completed the MVP by implementing Dropdown functionality along with Tabs components. 
- Your code is clean and git workflow is fluid as usual, great job 👍